snap/snapcraft.yaml

```diff
@@ -0,0 +1,253 @@
+name: kata-containers
+version: 1.0.0
```

Comment:
Ideally, this file would be `snap/snapcraft.yaml.in` and we've build it to avoid hard-coding all these versions numbers.

I'm also wondering what we can do (if anything) to avoid the script sections from being hard-coded.

Any thoughts @jcvenegas?

Final question - can we attempt to build this snap package under Ubuntu CI systems to ensure we haven't borked it when it's updated?

Reply:
> Final question - can we attempt to build this snap package under Ubuntu CI systems to ensure we haven't borked it when it's updated?

yes, we can :)